.eslintrc.js

```diff
@@ -48,5 +48,6 @@ module.exports = {
     '@sentry-internal/sdk/no-optional-chaining': 'off',
     '@sentry-internal/sdk/no-nullish-coalescing': 'off',
     '@sentry-internal/sdk/no-class-field-initializers': 'off',
+    'deprecation/deprecation': 'off',
```

Comment:
Are there many deprecation warnings? (More than ~20?)

If so this fine, but if it's not too many I would prefer disabling it locally to make it obvious which APIs will need to be updated in `sentry-capacitor` and the examples.

But let's keep it for next PR.